com.unity.ml-agents/Runtime/Demonstrations/DemonstrationRecorder.cs

```diff
@@ -12,19 +12,33 @@ namespace MLAgents
     [AddComponentMenu("ML Agents/Demonstration Recorder", (int)MenuGroup.Default)]
     public class DemonstrationRecorder : MonoBehaviour
     {
+        /// <summary>
+        /// Whether or not to record demonstrations.
+        /// </summary>
         [Tooltip("Whether or not to record demonstrations.")]
         public bool record;
 
-        [Tooltip("Base demonstration file name. Will have numbers appended to make unique.")]
+        /// <summary>
+        /// Base demonstration file name. If multiple files are saved, the additional filenames
+        /// will have a unique number appended.
```

Comment:
Should we call out that it is a sequence?

Reply:
👍